src/main.py

```diff
@@ -31,9 +30,7 @@ def outline(year, lang):
 @app.route('/<lang>/<year>/contributors')
 @validate
 def contributors(year, lang):
-def contributors(year, lang):
-    contributors=contributors_util.get_contributors()
```

Comment:
Whoops. This commit totally rolls back the progress on this issue. Can you add these changes back in and only delete the duplicate `contributors` function definition above?

Reply:
🤦‍♀️ Oh my, I think I'm getting confused. It's tricky looking at the changes and suggestions with the green and the red!
Is this what we want?:
```
def contributors(year, lang):
    contributors=contributors_util.get_contributors()
    return render_template('%s/%s/contributors.html' % (lang, year), contributors=contributors, supported_languages=supported_languages)
```

Reply:
Yes, that's it! Also make sure to keep the "import contributors" bit above.

Reply:
Where is the "import contributors" bit?

Thanks for your patience!

Reply:
![image](https://user-images.githubusercontent.com/1120896/60386906-be82d680-9a69-11e9-86af-e8761eb9f069.png)

This part is required because the `contributors` method calls `contributors_util.get_contributors()`.